.travis.yml

```diff
@@ -203,6 +216,29 @@ jobs:
         - tar xvfz generate_secret-windows-amd64-${APP_VERSION}.tar.gz -C /tmp
         - /tmp/generate_secret.exe
 
+      after_failure:
+        - SLACK_TEXT="${APP_VERSION} $TRAVIS_OS_NAME assets failed verification." send_to_slack.sh
+
+    - stage: 'Publish (real)'
+      if: type = push AND tag IS present AND tag =~ /^v[0-9]+\.[0-9]+\.[0-9]+$/
+      name: publish and notify in slack
+      os: linux
+      env: APP_VERSION=${TRAVIS_TAG#v}
+
+      before_install:
+        - ./scripts/pull_travis_ci_tools.sh
+
+      install:
+        # installs hub to ~/bin
+        - $HOME/travisci-tools/release_github/install_hub.sh
+        - export PATH=$HOME/bin:$HOME/travisci-tools/slack:$PATH
+
+      before_script: skip
+
+      script:
+        - SLACK_TEXT="${APP_VERSION} all assets verified. Publishing https://github.com/optimizely/agent/releases/tag/${TRAVIS_TAG}" send_to_slack.sh
+        - hub release edit --draft=false -m "" ${TRAVIS_TAG}
```

Comment:
this is what turns a release (draft) into a real published release

Reply:
Can you drop documentation about this here or some place in README may be? https://developer.github.com/v3/repos/releases/#create-a-release

Reply:
done